01-hello-git/README.md

```diff
@@ -66,194 +67,191 @@ or receiving changes from other repositories.
 
 ## Exercises
 
-### Creating a Fork
 
-A _fork_ is a personal copy of a repository hosted by a provider such as
-[GitHub](https://github.com). Create a fork and clone it.
+### Installing Git
```

Comment:
These are more like "prerequisites" than exercises, can you create a new section above Exercises called "Prerequisites" and move this up there?

Reply:
Jonathon, in order to make the README.md file more clear, I have considered the following sections:
1. Introduction
2. Prerequisites
3. Cloning an existing Git repository
4. Committing a modified version of a file to the repository
5. Exercises

Please let me know if you have something specific in your mind.